lib/helpers/path-to-eslint.js

```diff
@@ -0,0 +1,20 @@
+/*eslint-env node*/
+
+'use strict';
+
+var join = require('path').join;
+var asyncExec = require('./async-exec');
+
+/**
+ * Get the path to the ESLint executable in the dependencies
+ *
+ * @returns {Promise->String} the path to the ESLint executable
+ */
+module.exports = function() {
+  return asyncExec('npm', ['bin'])
+    .then(function(location) {
+      return join(
+        location.trim(), '../ember-cli-eslint/node_modules/.bin/eslint'
```

Comment:
In theory, `require.resolve('eslint')` will give you the location of eslint, then you can traverse the structure to find the associated bin file.

```
var eslintPath = path.join(require.resolve('eslint'), '../bin/eslint.js');
```

Maybe ^^ will work?